product_specifications/34DbH5/igsn.feature

```diff
@@ -0,0 +1,89 @@
+
+Feature: Repository exclusive for IGSN-DOIs
+
+        Scenario: Creating a IGSN Catalogue in Fabrica
+            Given that a user is creating a repository in Fabrica
+             When the user chooses the repository type
+             Then the  repository type field should show the 'IGSN Catalogue' option
+             When the user selects 'IGSN Catalogue'
+             Then a info message should be displayed reading: "This repository will only be able to mind IGSN-DOIs."
+             When clicking create a new repository with type 'IGSN Catalogue' should be created.
+ 
+        Scenario: Editing a IGSN Catalogue in Fabrica
+            Given the repository is type IGSN Catalogue
+             When the user is editing in the IGSN Catalogue edit form
+             Then the repository type field should be disabled.
+              And a tooltip should be shown when hovering over the field that read "IGSN Catalogues cannot be edited"
+
+        Scenario: Editing a repository in Fabrica
+            Given the repository is NOT type IGSN Catalogue
+             When the user is editing in the repository edit form
+             Then the  repository type form field should NOT show the 'IGSN Catalogue' option
+
+Feature: Creating IGSN-DOI from an IGSN Catalogue
+
+        Scenario: Creating/Editing a IGSN-DOI in any API
+            Given the user is creating a DOI for a IGSN Catalogue
+             When the user sends a payload for a DOI metadata
+             Then the metadata resourceTypeGeneral should be set to 'PhysicalObject' by default
+              And the record cretaed should include the field 'authority' set to 'igsn.org'
+ 
+        Scenario: Creating/Editing an IGSN-DOI in Fabrica form
+            Given the user is in the DOI form
+              And the user is within a IGSN Catalogue
+             When the user enter DOI metadata
+             Then the resourceTypeGeneral input field should be disabled
+              And a tooltip should be shown when hovering over the field that read "IGSN Catalogues can only mint PhysicalObject"
+              And the resourceTypeGeneral default should be set to 'PhysicalObject'
+              And the fabrica should set the metadata payload with the field 'authority' set to 'igsn.org'
+
+ 
+Feature: IGSN Catalogue DOI Transfers
```

Comment:
As I understood, they do want to be able to transfer IGSNs. Check with Cody please.

Reply:
i'll double check